docs/apply.md

```diff
@@ -17,9 +17,11 @@ A transform that accepts a custom template to execute. Must use the sql template
 ds = rasgo.get.dataset(id)
 
 ds2 = ds.apply(
-  sql='SELECT * FROM {{ source_table }} WHERE COLUMNVALUE = I17'
+  sql='SELECT * FROM {{ source_table }} WHERE COLUMNVALUE = I17',
+  source_table='db.schema.table'
 )
```

Comment:
So in `PyRasgo` the `source_table` here gets out-populated into the args dict like below, so it doesn't make sense to add that as a kwarg here for example code.

```python
args['source_table'] = ds.fqtn
```

I think it will be better to instead have an Apply transform with two/three jinja args. Mabe something like below
```python
ds2 = ds.apply(
  sql='SELECT * FROM {{ source_table }} WHERE {{ where_column }} = {{ where_value }}',
  where_column='NEW_LEADS_COL',
  where_value=100
)
```

Reply:
Good feedback thanks